dev/com.ibm.ws.jaxb_fat/test-applications/thirdPartyJaxbImplContextApp/src/jaxbimpl/thirdparty/web/ThirdPartyJAXBImplContextTestServlet.java

```diff
@@ -0,0 +1,222 @@
+/**
+ *
+ */
+package jaxbimpl.thirdparty.web;
```

Comment:
Should not have jar files checked into the repo.  Should be able to get them in build.gradle file if they are needed.  They are EclispeLink jars so they are available with persistence-3.1 so you should be able to use that feature in your server.xml if they are needed.